### ISSUE TITLE
OpenBlock Tanks

### ISSUE THREAD

**Issue by bookerthegeek** [SlimeKnights/TinkersConstruct#3407]
## Heyo Again ##
Trying to figure out what is going on. Since some mods seem to work yet others do not, I can not tell if this is an Tinkers issue, or an OpenBlocks issue. 

![ScreenShot](https://i.imgur.com/fIorwt8.png)

Seems any molten fluid added by Tinkers is rendering without a color inside the Openblocks tanks. Odd thing is that some mods and vanilla work fine, while others do not. 

### Versions ###
Minecraft 1.12.2
Forge 14.23.2.2624
CoFHCore 4.3.10.5
Industrial Craft 2 2.8.62-ex112
Mantle 1.3.1.21
OpenBlocks 1.7.6
OpenModsLib 0.11.5
Thermal Dynamics 2.3.10.4
Thermal Expansion 5.3.10.15
Thermal Foundation 2.3.10.6
Tinkers 2.9.1.65

**Comment by KnightMiner** [comment]
This is an OpenBlocks issue. We use fluid tinting and they apparently forgot to apply fluid tints to their fluids.

**Comment by bookerthegeek** [comment]
Makes sense, and I was thinking that at first, and will open this issue over at their issue tracker.

But a question then, why does thermal expansion, IC2, foundry and extra utilities fluids work? In the picture you can see that the IC2 UU Matter is working correctly. I would have thought that it was a all or nothing and not on a 'per mod' type of thing.

**Comment by TherminatorX** [comment]
They don't use fluid tinting
